samples/tab-sso/nodejs/.gitignore

```diff
@@ -63,5 +63,24 @@ dist/
 manifest/
```

Comment:
Better to commit the original env folder, refer to https://github.com/formulahendry/teamsfx-migrator/tree/main/templates/tab-sso/env